# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/tests/crashtests/crashtests.list
# Commit: a83043c7ab51
# Full Hash: a83043c7ab51233d38f3f65704de87aa3db3feac
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2023-07-10 09:40:14
# Description:
#   Bug 1825450 - Ensure device pixel scale for surfaces can't become infinite r=gfx-reviewers,lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D182429
# ==============================================================================

diff -r d3ca9e70625b -r a83043c7ab51 gfx/tests/crashtests/crashtests.list
--- a/gfx/tests/crashtests/crashtests.list	Sun Jul 09 21:06:35 2023 +0000
+++ b/gfx/tests/crashtests/crashtests.list	Sun Jul 09 21:22:10 2023 +0000
@@ -225,4 +225,5 @@
 load 1799495-1.html
 load 1802382-1.html
 load 1808830.html
+load 1825450.html
 